packages/web-react/src/components/UNSTABLE_EmptyState/demo/EmptyStateDefault.tsx

```diff
@@ -21,7 +21,9 @@ const EmptyStateDefault = () => (
           />
         </svg>
         <div className="docs-Placeholder__text">
-          <Heading size="xsmall">Placeholder</Heading>
+          <Text emphasis="bold" marginBottom="space-0">
+            Placeholder
+          </Text>
           <Text>Replace me with your own component</Text>
```

Comment:
This should be body-small-regular.
![image](https://github.com/user-attachments/assets/1a7dc0a0-6938-4387-a506-32a776ffde62)